common/persistence/cassandra/cassandraPersistenceTest.go

```diff
@@ -135,25 +137,25 @@ func (s *TestCluster) CreateSession() {
 	var err error
 	s.session, err = s.cluster.CreateSession()
 	if err != nil {
-		log.Fatal(`CreateSession`, err)
+		s.logger.Fatal("CreateSession", tag.Error(err))
 	}
 }
 
 // CreateDatabase from PersistenceTestCluster interface
 func (s *TestCluster) CreateDatabase() {
-	err := CreateCassandraKeyspace(s.session, s.DatabaseName(), 1, true)
+	err := CreateCassandraKeyspace(s.session, s.DatabaseName(), 1, false, s.logger)
```

Comment:
!

Reply:
just curious -- how does this make a difference? are you thinking test suits are dropping db from under each other (despite test suites seemingly generating a random db name for every run)?

Reply:
Don't ask. But apparently it is not a root of the problem.

Reply:
I am reverting it back.